lib/src/analyzers/lint_analyzer/rules/rules_list/avoid_returning_widgets/visit_declaration.dart

```diff
@@ -0,0 +1,72 @@
+part of 'avoid_returning_widgets.dart';
+
+Declaration? _visitDeclaration(
+  Declaration node,
+  SimpleIdentifier name,
+  TypeAnnotation? returnType,
+  Iterable<String> _ignoredNames,
+  Iterable<String> _ignoredAnnotations, {
+  required bool isSetter,
+}) {
+  final hasIgnoredAnnotation = node.metadata.firstWhereOrNull(
```

Comment:
@incendial you can use `any` instead `firstWhereOrNull`

Reply:
Fixed.